server/base/src/main/java/org/apache/accumulo/server/init/InitialConfiguration.java

```diff
@@ -91,9 +91,13 @@ public InitialConfiguration(Configuration hadoopConf, SiteConfiguration siteConf
     // Create a locality group that contains status so its fast to scan. When fate looks for work is
     // scans this family.
     initialFateTableConf.put(Property.TABLE_LOCALITY_GROUP_PREFIX.getKey() + "status",
-        FateSchema.TxColumnFamily.STR_NAME);
+        FateSchema.StatusColumnFamily.STR_NAME);
     initialFateTableConf.put(Property.TABLE_LOCALITY_GROUPS.getKey(), "status");
 
+    initialFateTableConf.put(Property.TABLE_LOCALITY_GROUP_PREFIX.getKey() + "reservation",
+        FateSchema.ReservationColumnFamily.STR_NAME);
+    initialFateTableConf.put(Property.TABLE_LOCALITY_GROUPS.getKey(), "reservation");
+
```

Comment:
This creates two separate locality groups. The value for the property set would need to be a comma separated list of the two families so status and reservation share the same locality group. (See code above this that sets other locality groups).